UnityProject/Assets/Scripts/Items/Medical/Genetics/MutationGrabber.cs

```diff
@@ -0,0 +1,108 @@
+using System.Collections;
+using System.Collections.Generic;
+using Systems.MobAIs;
+using UnityEngine;
+
+public class MutationGrabber : MonoBehaviour, IExaminable , ICheckedInteractable<PositionalHandApply>
```

Comment:
I just noticed, why are these things not namespaced?

Reply:
Just never did it

Reply:
namespace things, it will make whoever decides to try and setup assembly definitions for us's life much easier.

Reply:
ok cool

Reply:
humm, since I'm just auto generating the namespace for like they could do the same thing so I'm going to leave it for now